layers/+emacs/org/packages.el

```diff
@@ -547,6 +548,39 @@ Headline^^            Visit entry^^               Filter^^                    Da
 (defun org/init-ox-reveal ()
   (spacemacs|use-package-add-hook org :post-config (require 'ox-reveal)))
 
+(defun org/init-org-journal ()
+  (use-package org-journal
+    :defer t
+    :config
+    :init
+    (progn
+
+       (setq org-journal-dir "~/org/journal/")
+       (setq org-journal-file-format "%Y-%m-%d"))
+
```

Comment:
Oh, so this `org-journal-dir` and `org-journal-file-format` are my own preferences, they seem to have snuk in. Shall I remove those so that they can be set in user-config?

The default value for org-journal-dir is `~/Documents/journal/` and the default file format is `YYYYMMDD`